decentraland/screens/helpers.js

```diff
@@ -85,14 +88,16 @@ export const fundAccount = async accountAddress => {
     TEN
   );
   await transferManaAction.waitForNonceToUpdate();
+  onSuccess();
 };
 
-// TODO: Use functions/components properly
+// More about Toast component: https://docs.nativebase.io/Components.html#toast-def-headref
+const showToast = msg => Toast.show({ text: msg, duration: 3000 });
 export const showFatalError = msg => console.error(msg);
-export const showError = msg => console.warn(`ERROR: ${msg}`);
-export const showWarn = msg => console.warn(`WARN: ${msg}`);
-export const showInfo = msg => console.warn(`INFO: ${msg}`);
-export const showSuccess = msg => console.warn(`SUCCESS: ${msg}`);
+export const showError = msg => showToast(`ERROR: ${msg}`);
+export const showWarn = msg => showToast(`WARN: ${msg}`);
+export const showInfo = msg => showToast(`INFO: ${msg}`);
+export const showSuccess = msg => showToast(`SUCCESS: ${msg}`);
```

Comment:
Thoughts on `buttonText: 'Okay'` for dismissing in some cases?

Reply:
I've just added this option to all cases, there is a case that shouldn't have that?

Reply:
Thanks. I think the only time we wouldn't want it to be dismissible is if the toast were going to be around until something was complete. But I don't think a toast is the right UI element for that type of situation. So I think having it in all cases is fine.

Reply:
Resolving this